stage4/src/Controllers/HomeController.cs

```diff
@@ -10,6 +10,7 @@ namespace opensourceci.Controllers
     public class HomeController : Controller
     {
         private INameService nameService = new NameService();
+        private int unusedProperty = 0;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make 'unusedProperty' 'readonly'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS2933)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused private field 'unusedProperty'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1144)